controllers/git_tekton_resources_renovater_test.go

```diff
@@ -75,11 +77,30 @@ var _ = Describe("Git tekton resources renovater", func() {
 				repositories := generateRepositories(installedRepositoryUrls)
 				return []github.ApplicationInstallation{generateInstallation(repositories)}, "slug", nil
 			}
-			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{gitURL: "https://github/test/repo1"}))
+			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{componentKey: types.NamespacedName{Name: "testtrigger"}, gitURL: "https://github/test/repo1"}))
 			createBuildPipelineRunSelector(defaultSelectorKey)
 			Eventually(listJobs).WithArguments(buildServiceNamespaceName).WithTimeout(10 * time.Second).Should(HaveLen(1))
 			deleteComponent(componentNamespacedName)
 		})
+
+		It("It should trigger job, use default branch and default installation per job", func() {
+			installedRepositoryUrls := []string{
+				"https://github/test/repo1",
+				"https://github/test/repo2",
+			}
+			github.GetAppInstallations = func(appIdStr string, privateKeyPem []byte) ([]github.ApplicationInstallation, string, error) {
+				repositories := generateRepositories(installedRepositoryUrls)
+				return []github.ApplicationInstallation{generateInstallation(repositories)}, "slug", nil
+			}
+			componentsData := getComponentData(componentConfig{componentKey: types.NamespacedName{Name: "testdefault"}, gitURL: "https://github/test/repo1"})
+			componentsData.Spec.Source.GitSource.Revision = ""
+			componentNamespacedName := createComponentForPaCBuild(componentsData)
+			os.Setenv(InstallationsPerJobEnvName, "0")
+			createBuildPipelineRunSelector(defaultSelectorKey)
+			Eventually(listJobs).WithArguments(buildServiceNamespaceName).WithTimeout(10 * time.Second).Should(HaveLen(1))
```

Comment:
Why not to use globally defined `timeout` constant?

Reply:
10 seconds was there for all other tests,

changed it everywhere to 'timeout'